prvocisla.cpp

```diff
@@ -49,20 +52,26 @@ int * sieve(int n){
 
 void printPrime(int* L, int n) {
     int j = 0;
+
     while (L[j] <= n){
         cout << L[j] << " ";
```

Comment:
```suggestion
        cout << L[j] << "\n";
```
btw. mas dve cteni na jednu promennou... zbytecne.. ale to se asi vyoptimalizuje... kazdopdne pouzij valgrind! tusim chyby :/ ale ,uzu se plest

Reply:
jaktoze mam dve cteni?

Reply:
valgrindem to proslo - no leak

Reply:
> jaktoze mam dve cteni?

jak pisu, asi  se to beztak vyoptimalizuje, pro tvoje potreby je to jedno, ale je asi dobre to vedet ... IDK --- 

 obecne ` L[j]` je pristup do pole = cteni z pole (dereferencuje se pointer)

Reply:
> valgrindem to proslo - no leak

spis jsem mel strach o cteni neinicializovanych hodnot... ... napr. s tim `numPrime`... taky to zkontroluj pro vicero `n`... ktere muzou a nemusi byt prvocisla ... napr. 6, 7, 10, 11, 7, 100, 333, 4444, 1221, 777, 65321, 65327

Reply:
(to ze ti to projde na jednom vstupu neznamena ze to projde na vsech vstupech)